plugins/ern_v0.14.0+/react-native-fast-image_v5.3.0+/config.json

```diff
@@ -2,10 +2,17 @@
   "android": {
     "root": "android",
     "dependencies": [
-      "com.github.bumptech.glide:glide:4.7.1",
-      "com.github.bumptech.glide:annotations:4.7.1",
-      "annotationProcessor:com.github.bumptech.glide:compiler:4.7.1",
-      "com.github.bumptech.glide:okhttp3-integration:4.7.1"
+      "com.github.bumptech.glide:glide:4.12.0",
+      "com.github.bumptech.glide:annotations:4.12.0",
+      "annotationProcessor:com.github.bumptech.glide:compiler:4.12.0",
+      "com.github.bumptech.glide:okhttp3-integration:4.9.0"
+    ],
+    "replaceInFile": [
+      {
+        "path": "lib/build.gradle",
+        "string": "    sourceSets.main {",
+        "replaceWith": "    sourceSets.main {\n        java {\n            exclude \"**/FastImageGlideModule.java\" \n        }"
```

Comment:
How are we excluding the Glide Module? because this is not similar to what is mentioned in the link you sent

Reply:
They've created that flag to actually do what I'm doing here https://github.com/DylanVann/react-native-fast-image/blob/86edd7ffc54a293cd732801fc70c0d4ada03899c/android/build.gradle#L34.